environment.yml

```diff
@@ -1,17 +1,15 @@
 name: navo-env
 channels:
-  - conda-forge
+  - astropy
 dependencies:
-  - python=3.7
+  - python=3.8
   - numpy
   - astropy>=4.1
```

Comment:
I'm wondering a couple things about the astropy version, though not suggesting any change at this point:

- Currently this gives me v4.3.1, so we could up the minimum version to 4.3.1 as well.
- Version 5.0 may be released relatively soon.  I suspect we'll be fine with that, but we should definitely retest if/when that happens.

Reply:
thanks TomD, we can watch the astropy version and re-test.